docs/content/1.guide/7.plugins.md

```diff
@@ -27,6 +27,38 @@ export default defineNitroPlugin((nitroApp) => {
 })
 ```
 
+**Example:** Development Plugin
+
+You can use plugins to run development tasks, such as database migrations.
+
+```ts
+import { migrate } from 'drizzle-orm/better-sqlite3/migrator'
```

Comment:
Let's remove migration as an example. it is not best usage of plugins and i am thinking to introduce a better way for them.

Reply:
As said before, we are working on a better solution which is not plugins. There is no best place to advice today.

Reply:
Removed it